modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/Application.java

```diff
@@ -24,7 +24,6 @@
  * Web console launcher.
  */
 @SpringBootApplication
-@EnableScheduling
```

Comment:
Not sure if it is good idea to move this annotation to service.
We may need scheduling in other places later.
I would keep it here.

Reply:
I think will be better to create separate configuration.